includes/admin/settings/class-wc-admin-settings-woorule.php

```diff
@@ -124,7 +123,6 @@ public static function order_status_changed($id, $status = '', $new_status = '')
                           'subtotal' => $item->get_total()
                           );
 
-                        // this is bullshit
```

Comment:
is it stopped to be a bullshit? 😁

Reply:
@IlyaKobus I think it was related to another part of code that was changed with time 😄